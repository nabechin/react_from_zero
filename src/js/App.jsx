import React from 'react';
import styles from '../css/app.css';

const App = () => <h2 className={styles.header}>hello world</h2>;
export default App;
