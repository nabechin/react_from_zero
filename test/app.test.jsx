import React from 'react';
import renderer from 'react-test-renderer';
import App from '../src/js/App';

describe('<App>', () => {
  test('render correctly', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
