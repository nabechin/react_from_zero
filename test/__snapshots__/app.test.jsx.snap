// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App> render correctly 1`] = `
<h2
  className="header"
>
  hello world
</h2>
`;
